Code/educopter_frontend/lib/create_mission/view/create_mission.dart

```diff
@@ -34,176 +34,231 @@ class _MissionCreateScreenState extends State<MissionCreateScreen> {
+                        //   },
+                        //   child: Text('Filter steden'),
+                        // ),
+                        SizedBox(width: 10),
+                      ],
                     ),
                   ],
                 ),
               ),
-              DropdownButton(
-                value: dropdownvalue,
-                icon: const Icon(Icons.keyboard_arrow_down),
-                items: availableWorldmaps.map((Worldmap map) {
-                  return DropdownMenuItem(
-                    value: map.mapName,
-                    child: Text(map.mapName),
-                  );
-                }).toList(),
-                onChanged: (String? selectedValue) {
-                  setState(() {
-                    dropdownvalue = selectedValue!;
-                  });
-                },
+              SizedBox(height: 20),
+              Row(
+                mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                children: [
+                  ElevatedButton(
+                    onPressed: () {},
+                    child: Text("Voeg alle steden toe"),
+                  ),
+                  ElevatedButton(
+                    onPressed: () {},
+                    child: Text("Wis lijst geselecteerd steden"),
+                  ),
+                  ElevatedButton(
+                    onPressed: () {},
+                    child: Text("Sla missie op"),
+                  ),
+                ],
               ),
-            ],
-                ),
-          ),
-          FutureBuilder(
-            future: processCsv(context),
-            builder: ((context, snapshot) {
-              if (snapshot.connectionState == ConnectionState.done) {
-                List<City> allCitiesFromMap = snapshot.data as List<City>;
-                List<City> filteredCities =
-                    completeFilter.meetCriteria(allCitiesFromMap);
-                print(filteredCities[5].cityName);
+              SizedBox(height: 20),
+              Row(
+                children: [
+                  SizedBox(width: 10),
+                  Expanded(
+                    child: FutureBuilder(
+                      future: processCsv(context),
+                      builder: ((context, snapshot) {
+                        if (snapshot.connectionState == ConnectionState.done) {
+                          List<City> allCitiesFromMap =
+                              snapshot.data as List<City>;
+                          List<City> filteredCities =
+                              completeFilter.meetCriteria(allCitiesFromMap);
+                          print(filteredCities[5].cityName);
 
-                //Ik wil iets zeggen als...
-                //Criteria totalFilter = createFilter(filterCriteria);
-                //en dan...
-                //List<City> filteredCities =
-                //    totalFilter.meetCriteria(allCitiesFromMap);
+                          //Ik wil iets zeggen als...
```

Comment:
Volgens mij is dit al gelukt?

Reply:
Nee, de methode om een completefilter te schrijven moet ik nog implementeren. Deze is nu alleen hardcoded zonder logica.